sheets/3/exercise-2.tex

```diff
@@ -71,10 +71,18 @@
 
 \begin{enumerate}\setcounter{enumi}{2}
 	\item
-		Lasse Neo solange laufen bis sich mindestens eine Münze auf dem Feld unter ihm befindet.
+		Lass Neo immer wenn sich keine Münze unter ihm befindet einen Schritt nach vorne gehen.
+		Tipp: Verwende hierfür am besten eine Endlosschleife und Schreibe den Code für alle Unteraufgaben dieser Aufgabe in den Schleifenkörper.
+		Eine Endlosschleife kann z.B. so aussehen:

```

Comment:
Vielleicht eine Infobox, dass Endlosschleifen wirklich endlos sind und man daher in richtigen programmen immer aufpassen muss, dass man die rigendwann auch abbricht. (ist aber nicht super wichtig für den Vorkurs, da die Tasks beim schließen des Fensters von Außen abgebrochen werden auch bei endlosschleifen.)